### PR TITLE
Added Image

### PR DESCRIPTION
**Pull request by mriganksingh1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@
 - Different types of merging code in GitHub namely:
   - Merge Commit
   - Rebase and Merge
-  - Squash and Merge
+  - Squash and Merge  
+  <a href="https://drive.google.com/uc?export=view&id=1TqQEmimySSZtcFO2tWBBEdgGmlDUizZr"><img src="https://drive.google.com/uc?export=view&id=1TqQEmimySSZtcFO2tWBBEdgGmlDUizZr" style="width: 650px; max-width: 100%; height: auto" title="Click to enlarge picture" />
```
Comment (praveenscience) [inline README.md]: Let's use imgur.com for image hosting or have the image here.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Would you like to do it or shall I do it?

File: README.md (hunk 1 above)
Comment (mriganksingh1) [reply README.md]: okay i will see imgur.com

Comment (praveenscience) [reply README.md]: Waiting! 😁

Comment (praveenscience) [reply README.md]: @mriganksingh1 I'll update it.

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**